robocop/checkers/tags.py

```diff
@@ -46,10 +46,22 @@
     "0603": Rule(
         rule_id="0603",
         name="tag-with-reserved",
```

Comment:
BTW What about changing its name to `tag-with-reserved-word`? I don't know why wanted to keep it so short, that it doesn't make sense.

Reply:
I agree. ``tag-with-reserved-word`` sounds better